spec/bojangles_spec.rb

```diff
@@ -131,7 +135,7 @@
       before(:each) { FileUtils.rm 'error_messages.json' }
       it 'returns an empty array' do
         expect(File.file?('error_messages.json')).to be false
-        expect(Bojangles.cached_error_messages).to eql []
+        expect(Bojangles.cached_error_messages).to eql({})
```

Comment:
Do you need the parenthesis here?

Reply:
Yes, otherwise the curly braces get interpreted as a block argument to `eql` rather than a Hash literal.